Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "winit"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
Please do not bump the version! But please add an entry in the changelog instead.